rlwe/params.go

```diff
@@ -88,14 +91,20 @@ func NewParameters(logn int, q, p []uint64, sigma float64, ringType ring.Type) (
 // NewParametersFromLiteral instantiate a set of generic RLWE parameters from a ParametersLiteral specification.
 // It returns the empty parameters Parameters{} and a non-nil error if the specified parameters are invalid.
 func NewParametersFromLiteral(paramDef ParametersLiteral) (Parameters, error) {
+
```

Comment:
We should put somewhere what fields are mandatory or can be left empty, and what happens if done so

Reply:
Yes. If we agree with the design, I'll make a pass on the documentation to reflect that.

Reply:
Done.